administrator/language/en-GB/plg_workflow_notification.ini

```
@@ -13,7 +14,8 @@ PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_MSG="Title: '%1$s', Changed by '%2$s',
 PLG_WORKFLOW_NOTIFICATION_RECEIVERS_LABEL="Users"
 PLG_WORKFLOW_NOTIFICATION_RECEIVERS_SELECT="Select single  Users who obtain a Notification"
 PLG_WORKFLOW_NOTIFICATION_SENDMAIL_LABEL="Send Notification"
-PLG_WORKFLOW_NOTIFICATION_SENT="Notifications have been sent"
+PLG_WORKFLOW_N_NOTIFICATION_SENT="%d notifications have been sent"
+PLG_WORKFLOW_N_NOTIFICATION_SENT_1="A notification has been sent"
```

```suggestion
PLG_WORKFLOW_N_NOTIFICATION_SENT="%d notifications sent."
PLG_WORKFLOW_N_NOTIFICATION_SENT_1="Notification sent."
```
To be consistent with other language strings.

---

Lline 17 ok. 
Not sure about the second. The user can define many recipients for a message. But for different reasons, some of these users are filtered, so that only one (1) message is sent.

---

Your version doesn't convey that to me. I interpret them to be the same.

---

@chmst from what you said it suggests to me that its possible for all of the users to be filtered. What is displayed if that happens?